icu4j/build.properties

```diff
@@ -4,9 +4,9 @@
 #* Copyright (C) 2009-2016, International Business Machines Corporation and    *
 #* others. All Rights Reserved.                                                *
 #*******************************************************************************
-api.report.version = 68
-api.report.prev.version = 67
-release.file.ver = 68_2
-api.doc.version = 68.2
-maven.pom.ver = 68.2
+api.report.version = 69
+api.report.prev.version = 68
+release.file.ver = 69_1
+api.doc.version = 69.1 Release Candidate
```

Comment:
Sorry to keep nitpicking, but based on the docs/previous version updates I think we want this for RC:
```
release.file.ver = 69rc
api.doc.version = 69 Release Candidate
```

Reply:
Good point, sorry for missing this as well. 
(For example, here's what ICU 67 RC had: https://github.com/unicode-org/icu/commit/73b50e6463cb68d2c0adfff3b081a995dfc1db42#diff-c2d7ab23228bfdfbeae4c9e6184d9e63cb25d10f6690f75f3aa40f44f7df7635R10)

Reply:
Ah thanks for noticing this :)